src/rotation_conversions.jl

```diff
@@ -3,7 +3,7 @@ import Base: convert
 
 
 # a sgn function
-@inline sgn{T}(x::T) = copysign(T(1), x)  #hmm, x = -0.0
+@inline sgn{T}(x::T) = copysign(one(T), x)  # hmm, x = -0.0
```

Comment:
Ah I see.  Could do with a comment saying that it's intentionally different from `Base.sign` since `sgn(0) == 1` and `sgn(-0) = -1`